docs/tutorials/toy-connector.md

```diff
@@ -98,7 +100,7 @@ We'll use each of these files later. But first, let's write some code!
 In the connector package directory, create a single python file `source.py` that will hold our implementation:
 
 ```bash
-$ touch source.py
```

Comment:
removed `$` for any of the one line commands. in these cases copying and pasting is nice. this was explicitly asked for by user.